gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/ValidationErrorMessage.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import io.camunda.zeebe.protocol.record.intent.UserTaskIntent;
+
+public class ValidationErrorMessage {
```

Comment:
❌ We are not using this class anymore, right? Let's remove it.

Reply:
Missed that, thank you!